README.md

```diff
@@ -78,64 +77,16 @@ Table of contents
     pip install -v -e .
     ```
 
-## Contributing
+## 🤔 Contributing
 
 See the [Contributing Guide](./contributing.md).
 
 You may consider starting or joining in with a [discussion](https://github.com/alan-turing-institute/scivision/discussions), or opening an [issue](https://github.com/alan-turing-institute/scivision/issues).
 
-## Documentation
+## 📚 Documentation
 
-Documentation is available at https://scivision.readthedocs.io/en/latest/
+User documentation for scivision is available at our [readthedocs](https://scivision.readthedocs.io/en/latest/) website.
```

Comment:
Just 'Documentation' seems more natural to me (rather that 'User documentation')